test-configs.yaml

```diff
@@ -1689,18 +1696,32 @@ test_configs:
   - device_type: imx6sx-sdb
     test_plans:
       - baseline
+      - baseline-nfs
 
   - device_type: imx6ul-14x14-evk
     test_plans:
       - baseline
+      - baseline-nfs
 
   - device_type: imx6ull-14x14-evk
     test_plans:
       - baseline
+      - baseline-nfs
 
   - device_type: imx7d-sdb
     test_plans:
       - baseline
+      - baseline-nfs
+
+  - device_type: imx7ulp-evk
```

Comment:
And it looks like this one is also getting duplicated, can you please check the patch?

Maybe it happened as part of the rebase.

Reply:
Hi gctucker,
This was a rebase issue. I have corrected it. Sorry for the inconvenience.